src/game_entities/entity.py

```diff
@@ -63,10 +64,13 @@ def __str__(self) -> str:
         Return the formatted text version of the entity based on its name,
         underscores are replaced by spaces and numbers are removed.
         """
-        string_entity: str = self.name.replace("_", " ").title()
-        string_entity = re.sub(
-            r"[0-9]+", "", string_entity
-        )  # Remove numbers like the id
+        try:
+            string_entity: str = dict_entity_names[re.sub(r"[0-9]+", "", self.name).lower()]
+        except KeyError:
+            string_entity: str = self.name.replace("_", " ").title()
+            string_entity = re.sub(
+                r"[0-9]+", "", string_entity
```

Comment:
Maybe we should extract this regex somewhere, since it's used twice.

Reply:
Done.